staging/src/k8s.io/apiserver/pkg/cel/environment/base.go

```diff
@@ -173,7 +173,7 @@ var baseOptsWithoutStrictCost = []VersionedOptions{
 	{
 		IntroducedVersion: version.MajorMinor(1, 32),
 		EnvOptions: []cel.EnvOption{
-			UnversionedLib(ext.TwoVarComprehensions),
+			ext.TwoVarComprehensions(),
```

Comment:
Just hit this incompatibility in another project: https://github.com/kubernetes/kubernetes/issues/129869

Seems a simple fix but I'd prefer not to wait until Kubernetes 1.33 to update cel-go... Can we backport this change? Or is another patch of cel-go that fixes the incompatibility more feasible?

Reply:
@lambdanis Are you looking for a backport of versioning for CEL libraries here or something else? This would be the desired PR if so: https://github.com/google/cel-go/pull/1075

Reply:
@lambdanis Is Kubernetes 1.33.0-alpha.1(release on Feb 4th, 2025) an option to you? If so, this pr could catch the alpha1 release